<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Users">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QTableWidget" name="UsersTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>870</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Computers">
     <attribute name="title">
      <string>Computers</string>
     </attribute>
     <widget class="QTableWidget" name="ComputersTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>870</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mostly used window</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Second mostly used window</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Third mostly used window</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mostly used window percent</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Second mostly used window percent</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Third mostly used window percent</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Currently active window</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number of opened windows</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disk memory usage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Min CPU frequency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max CPU frequency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Current CPU frequency</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>System boot time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total memory used</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Applications">
     <attribute name="title">
      <string>Applications</string>
     </attribute>
     <widget class="QTableWidget" name="ApplicationsTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1600</width>
        <height>870</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Application name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Create time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RSS memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>VMS memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Shared memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
